src/main/java/wolox/training/models/dto/BookDTO.java

```diff
@@ -0,0 +1,52 @@
+package wolox.training.models.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Objects;
+import lombok.Data;
+import org.springframework.util.CollectionUtils;
+
+@Data
+public class BookDTO {
+
+    private String isbn;
+    private String title;
+    private String subtitle;
+    private List<DataName>  publishers;
+    @JsonProperty("publish_date")
+    private String publishDate;
+    @JsonProperty("number_of_pages")
+    private Integer numberOfPages;
+    private List<DataName> authors;
+    private List<DataName> subjects;
+    private Cover cover;
+
+    public String getPublishers(){
+        return !CollectionUtils.isEmpty(this.publishers) ? this.publishers.get(0).getName() : null;
```

Comment:
You can flip the validation so you don't compare by the negative: 

`CollectionUtils.isEmpty(this.publishers) ? null : this.publishers.get(0).getName()`

Reply:
Complementing Jime's suggestion, the best approach is to start with the affirmative condition, so the code is more readable.